doc/source/security.md

```diff
@@ -159,9 +159,19 @@ There's ongoing work on making this easier!
 
 ## Kubernetes Network Policies
 
-Kubernetes has optional support for [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) which lets you restrict how pods can communicate with each other and the outside world. This can provide additional security within JupyterHub, and can also be used to limit network access for users of JupyterHub. By default, the JupyterHub helm chart **disables** network policies.
-
-**Important**: If you decide to enable network policies, you should be aware that a Kubernetes cluster may have partial, full, or no support for network policies. Kubernetes will **silently ignore** policies that aren't supported. Please use **caution** if enabling network policies and verify the policies behave as expected, especially if you rely on them to restrict what users can access.
+Kubernetes has optional support for [network
+policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+which lets you restrict how pods can communicate with each other and the outside
+world. This can provide additional security within JupyterHub, and can also be
+used to limit network access for users of JupyterHub. By default, the JupyterHub
```

Comment:
Let's make the sentence "By default..." be its own paragraph so it stands out to the reader.